common/ledger/blkstorage/blockstore.go

```diff
@@ -73,19 +74,27 @@ func (store *BlockStore) RetrieveTxByID(txID string) (*common.Envelope, error) {
 	return store.fileMgr.retrieveTransactionByID(txID)
 }
 
-// RetrieveTxByID returns a transaction for given transaction id
+// RetrieveTxByBlockNumTranNum returns a transaction for the given <blockNum, tranNum>
 func (store *BlockStore) RetrieveTxByBlockNumTranNum(blockNum uint64, tranNum uint64) (*common.Envelope, error) {
 	return store.fileMgr.retrieveTransactionByBlockNumTranNum(blockNum, tranNum)
 }
 
+// RetrieveBlockByTxID returns the block for the specified txID
 func (store *BlockStore) RetrieveBlockByTxID(txID string) (*common.Block, error) {
 	return store.fileMgr.retrieveBlockByTxID(txID)
 }
 
+// RetrieveTxValidationCodeByTxID returns the validation code for the specified txID
 func (store *BlockStore) RetrieveTxValidationCodeByTxID(txID string) (peer.TxValidationCode, error) {
 	return store.fileMgr.retrieveTxValidationCodeByTxID(txID)
 }
 
+// ExportTxIds writes all the TxIDs to the writer. Technically, the TxIDs appear in the sort order of radix-sort/shortlex. However,
+// since practically all the TxIDs are of same length, so the sort order would be the lexical sort order
+func (store *BlockStore) ExportTxIds(writer io.Writer) error {
+	return store.fileMgr.index.exportUniqueTxIDs(writer)
```

Comment:
Not related to this PR. We can, in fact, merge `fileMgr` methods with the `BlockStore` structs. As a result, we can avoid most of the indirections used in this file. Further, this line would become `store.index.exportUniqueTxIDs()`.

Reply:
Yes, we can revisit this. But this does not bother much after having removed the interfaces in this package.